mail_tracking/controllers/main.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import werkzeug
+from psycopg2 import OperationalError
+from openerp import api, http, registry, SUPERUSER_ID
+import logging
+_logger = logging.getLogger(__name__)
+
+BLANK = 'R0lGODlhAQABAIAAANvf7wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=='
+
+
+class MailTrackingController(http.Controller):
+
+    def _request_metadata(self):
+        request = http.request.httprequest
+        return {
+            'ip': request.remote_addr,
+            'user_agent': request.user_agent,
+            'os_family': request.user_agent.platform,
+            'ua_family': request.user_agent.browser,
+        }
+
+    @http.route('/mail/tracking/open/<string:db>'
+                '/<int:tracking_email_id>/blank.gif',
+                type='http', auth='none')
+    def mail_tracking_open(self, db, tracking_email_id, **kw):
```

Comment:
OK, there's another way to do this. The menus could go into *Messaging* for users belonging to *Mail tracking* group*. Surely somebody will need to review this, and giving them admin & technical features privileges is too much for the job. They just need to see the reason of rejection, for instance to delete an email address or to call the customer to tell information manually.

And, besides, [right now all users have read access](https://github.com/OCA/social/pull/67/files/129ed1ba89cff54fcee0ab538f2c20093031cea5#diff-104f5a6b8a70f361300607b4137d4673R2).